src/utils/client.js

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import 'unfetch/polyfill';
```

Comment:
Bah, linking it in the main file makes the linter complain that it's imported but never used... guessing linting is done pre-rollup in our build? Will try and fix